unsafeslice.go

```diff
@@ -82,6 +82,14 @@ func ByteSliceFromInt16Slice(b []int16) []byte {
 	return *(*[]byte)(newSliceHeader(unsafe.Pointer(&b[0]), len(b)*Uint16Size))
 }
 
+func Uint8SliceFromByteSlice(b []byte) []uint8 {
```

Comment:
These didn't exist because `byte` is a `type byte = uint8` alias for uint8. That said it doesn't seem unreasonable to add them for completeness, but these should just `return b`.

Reply:
done